components/TravelMethodButtons/TravelMethodButtons.jsx

```diff
@@ -0,0 +1,50 @@
+import { Box, Button, Text, SimpleGrid, Icon, Flex } from '@chakra-ui/react';
+
+export function TravelMethodButtons({
```

Comment:
can we rename this function as `TravelMethodButtonsContainer` please? :pray: 

```suggestion
export function TravelMethodButtonsContainer({
```

Reply:
This is resolved